metadata-etl/src/main/resources/jython/HiveExtract.py

```diff
@@ -158,16 +202,44 @@ def format_table_metadata_v2(self, rows, schema):
         # sort the field_list by IntegerIndex
         field_list = sorted(field_list, key=lambda k: k['IntegerIndex'])
         # process the record of table
-        table_record = {TableInfo.table_name: row_value[1], TableInfo.type: 'Table', TableInfo.serialization_format: row_value[2],
+
+        if row_value[20].lower() == 'dali':
+          urn = 'dalids:///' + row_value[0] + '/' + row_value[18]
+          instance_record = DatasetInstanceRecord(row_value[25],
+                                      long(self.db_id),
+                                      'grid',
+                                      'eat1',
```

Comment:
please get the value by looking up the cfg_database based on db_id.

when db_id=65, the value should probably be ltx1, ltx1-holdem, ...